python/amici/sbml_import.py

```diff
@@ -1385,12 +1385,12 @@ def _reduce_stoichiometry(self, species_solver, volume_updates) -> List:
             List of species indices which remain later in the ODE solver
 
         :param volume_updates:
-            List with updates for the stoichiometrix matrix accounting for
+            List with updates for the stoichiometric matrix accounting for
             compartment volumes
 
         :returns volume_updates_solver:
             List (according to reduced stoichiometry) with updates for the
-            stoichiometrix matrix accounting for compartment volumes
+            stoichiometric matrix accounting for compartment volumes
```

Comment:
what was up there???

Reply:
Can only guess that somebody was watching Asterix while coding...